### PR TITLE
KHWE-15: ESLint setup

### PR DESCRIPTION
**Pull request by pavlobaranchuk**
[eslint-config-airbnb-base](https://www.npmjs.com/package/eslint-config-airbnb-base) package was chosen as the basis for ESLint configuration.
This package provides Airbnb's base JS .eslintrc (without React plugins) as an extensible shared config.

What's more, there are two quick fixes within the same pull request:

- Search field was moved to the header (that is logically correct)
- The old value for the font-size was returned(16px)

spotty

### CODE REVIEW

File: src/components/Main/Main.html [hunk 1]
```diff
@@ -4,31 +4,39 @@
...
+    <header class="mdc-top-app-bar app-bar">
+      <div class="main__header"></div>
+      <div class="main__search"></div>
+    </header>
     <main class="main__section main-content">
       <div class="main__content-mount mdc-top-app-bar--fixed-adjust">
-        <h5 class="mdc-typography--headline3">Albums</h5>
+        <h3 class="mdc-typography--headline4">Albums
+        </h3>
+        Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums
```
Comment (ihor4568) [inline src/components/Main/Main.html]: ???

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "private": true,
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack -p",
-    "start": "webpack-dev-server --mode=development"
+    "start": "webpack-dev-server --mode=development",
+    "eslint": "node_modules/.bin/eslint src/"
```
Comment (ihor4568) [inline package.json]: there is no need to write path inside npm scripts - so it will be just 'eslint src/'

Comment (ihor4568) [inline package.json]: please rename script to `lint`

File: package.json [hunk 3]
```diff
@@ -37,7 +42,12 @@
     }
   },
   "lint-staged": {
-    "*.{js,json,scss}": [
+    "*.js": [
+      "node_modules/.bin/eslint --max-warnings=0",
```
Comment (ihor4568) [inline package.json]: no need to write full path

**Comment by ihor4568** [comment]
Please use default sizes for fonts and icons in sidebar and top bar as in demos

image

File: src/components/Main/Main.html (hunk 1 above)
Comment (pavlobaranchuk) [reply src/components/Main/Main.html]: Just a sample content :-) I will remove it

**Comment by pavlobaranchuk** [comment]
spotty

**Review by ihor4568 — APPROVED** [review]

**Pull request merged**